test/libraries/Position.t.sol

```diff
@@ -21,4 +22,40 @@ contract PositionTest is Test {
         }
         assertEq(positionSlot, expectedPositionSlot, "slots not equal");
     }
+
+    function test_fuzz_update(
+        int128 liquidityDelta,
+        Position.Info memory pos,
+        uint256 newFeeGrowthInside0X128,
+        uint256 newFeeGrowthInside1X128
+    ) public {
+        Position.Info storage position = positions[0];
+        position.liquidity = pos.liquidity;
+        position.feeGrowthInside0LastX128 = pos.feeGrowthInside0LastX128;
+        position.feeGrowthInside1LastX128 = pos.feeGrowthInside1LastX128;
+
+        uint128 oldLiquidity = position.liquidity;
+
+        if (position.liquidity == 0 && liquidityDelta == 0) {
+            vm.expectRevert(Position.CannotUpdateEmptyPosition.selector);
+        }
+
+        // new liquidity cannot overflow uint128
+        uint256 newLiquidity = uint256(int256(uint256(position.liquidity)) + int256(liquidityDelta));
+        if (newLiquidity > type(uint128).max) {
+            vm.expectRevert(SafeCast.SafeCastOverflow.selector);
+        }
+
+        Position.update(position, liquidityDelta, newFeeGrowthInside0X128, newFeeGrowthInside1X128);
+        if (liquidityDelta == 0) {
+            assertEq(position.liquidity, oldLiquidity);
+        } else if (liquidityDelta > 0) {
+            assertGt(position.liquidity, oldLiquidity);
```

Comment:
can this not be `assertEq(position.liq, oldLiquidity + liquidityDelta)` and same with the `else` brach too